ocs_ci/ocs/ui/page_objects/buckets_tab.py

```
@@ -734,3 +750,69 @@ def navigate_to_bucket_permissions(self, bucket_name: str = None):
+        self.do_click(kebab_locator, enable_screenshot=True)
+        self.do_click(self.bucket_tab["object_action_preview"], enable_screenshot=True)
+
+        WebDriverWait(self.driver, timeout).until(
+            lambda d: len(d.window_handles) > len(original_handles)
+        )
+
+        new_handles = set(self.driver.window_handles) - original_handles
+        preview_handle = new_handles.pop()
+        self.driver.switch_to.window(preview_handle)
+        blob_url = self.driver.current_url
+        logger.info(f"Preview tab opened with URL: {blob_url}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)